README.md

```diff
@@ -1,9 +1,98 @@
-# README
+<a name="Hello-Rails-Bak-End"></a>
+Front-end react project link https://github.com/karam084/hello-react-front-end/pull/1
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
 
-Things you may want to cover:
+
+<div align="center">
+
+  <img src="./app/assets/images/recipe_erd_2_members.png" alt="logo" width="140"  height="auto" />
```

Comment:
- [ ] Kindly remove the microverse logo as it's not required.

Reply:
Thanks solved